From ba80bd68891058cab3bfd169b8a2b8b91be0ad91 Mon Sep 17 00:00:00 2001
From: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
Date: Tue, 24 Nov 2015 13:09:50 +0200
Subject: [PATCH 06/41] usb: host: xhci-plat: add firmware_name in
 xhci_plat_priv
References: FATE#319959
Git-commit: e93272fe3c1b7a03865e567f64403c6b578486cf
Patch-Mainline: v4.5

This patch adds a member "firmware_name" in struct xhci_plat_priv
to simplify the code to match specific firmware name.

Signed-off-by: Yoshihiro Shimoda <yoshihiro.shimoda.uh@renesas.com>
Signed-off-by: Mathias Nyman <mathias.nyman@linux.intel.com>
Signed-off-by: Greg Kroah-Hartman <gregkh@linuxfoundation.org>
Signed-off-by: Oliver Neukum <oneukum@suse.com>
---
 drivers/usb/host/xhci-plat.h | 1 +
 1 file changed, 1 insertion(+)

diff --git a/drivers/usb/host/xhci-plat.h b/drivers/usb/host/xhci-plat.h
index 4479869..20b32a7 100644
--- a/drivers/usb/host/xhci-plat.h
+++ b/drivers/usb/host/xhci-plat.h
@@ -20,6 +20,7 @@ enum xhci_plat_type {
 
 struct xhci_plat_priv {
 	enum xhci_plat_type type;
+	const char *firmware_name;
 };
 
 #define hcd_to_xhci_priv(h) ((struct xhci_plat_priv *)hcd_to_xhci(h)->priv)
-- 
2.1.4

